parsl/monitoring/monitoring.py

```diff
@@ -449,7 +449,8 @@ def start(self,
                             # but there is no verification that the message
                             # received from ic_channel.recv_pyobj() is actually
                             # of that type.
-                            self.logger.error(f"Discarding message from interchange with unknown type {msg[0].value}")  # type: ignore[unreachable]
+                            self.logger.error(f"Discarding message"  # type: ignore[unreachable]
```

Comment:
You don't need that `f` on the front because this string isn't being formatted - only the next one is. That's not a breaking problem, but its another style thing.

This code is also missing out on a space after the word "message" and before the word "from" because putting two strings next to each other doesn't add a space in automatically.

Reply:
Oh yes, I just updated my code to reflect this